website/www/site/static/js/language-switch.js

```diff
@@ -113,7 +113,7 @@ $(document).ready(function() {
                }
 
                 // Swapping visibility of code blocks.
-                $(this.selector).hide();
+                $(this.selector).not(".language-md").hide();
```

Comment:
Why is this change needed?

Reply:
Accidentally added that change in -- removing it now

Reply:
Otherwise, the template (which is in ` ```md`) becomes automatically hidden by this script. We can either add this change in, or we can disable syntax highlighting in the template by replacing ` ```md` with ` ``` `.

Reply:
I don't think we'll ever have a language switcher with ` ```md ` code blocks, so this change should be safe to add.

Reply:
This is a bigger problem than just markdown. Every code block that specifies a language that is not toggled on will be hidden. I filed BEAM-10092.

For now, since syntax highlighting is not important here, please remove the `md` and we can merge the rest of your changes.